.github/workflows/main.yml

```diff
@@ -19,9 +19,21 @@ jobs:
       - name: Static Analysis
         uses: pre-commit/action@v3.0.0
 
+  synth-test-dev:
+    needs: tests
+    uses: "./.github/workflows/synth-test.yml"
+    with:
+      CONTEXT: dev
+
+  synth-test-prod:
+    needs: tests
+    uses: "./.github/workflows/synth-test.yml"
+    with:
+      CONTEXT: prod
+
   cdk-deploy-dev:
     if: ${{github.ref == 'refs/heads/dev'}}
-    needs: tests
+    needs: [tests, synth-test-dev, synth-test-prod]
```

Comment:
hmm, why does this need `synth-test-prod`?  could we not deploy to dev account as long as `synth-test-dev` passes?

Reply:
Good point.  I have updated the PR accordingly.